### PR TITLE
Update alexa

### PR DESCRIPTION
**Pull request by Rexadev**

### CODE REVIEW

File: privacy/native/alexa [hunk 1]
```diff
@@ -1,4 +1,8 @@
 data.alexa.com
-
 device-metrics-us.amazon.com
 device-metrics-us-2.amazon.com
+alexa-smart-nudge.amazon.in
+alexa.a2z.com
+mshop-configs.shopping.alexa.a2z.com
```
Comment (rs) [inline privacy/native/alexa:6]: This one is covered by previous line

File: privacy/native/alexa [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+aws.a2z.com
```
Comment (rs) [inline privacy/native/alexa:7]: This will likely break non alexa stuff

File: privacy/native/alexa [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+redirect.prod.experiment.routing.cloudfront.aws.a2z.com
```
Comment (rs) [inline privacy/native/alexa:8]: Same here, does look like alexa specific

**Review by rs — CHANGES_REQUESTED** [review]

**Comment by romaincointepas** [comment]
Those may break some features.

**Pull request closed without merging**